crypto/schemes.go

```diff
@@ -171,10 +166,12 @@ const ShortSigSchemeID = "bls-unchained-on-g1"
 // "timelock encryption" as done by tlock.
 // This schemes has the group public key on G2, so 96 bytes, and the beacon signatures on G1, so 48 bytes.
 // This means databases of beacons produced with this scheme are almost half the size of the other schemes.
+// Deprecated: However this scheme is using the DST from G2 for Hash to Curve, which means it is not spec compliant.
 //
 //nolint:dupl
 func NewPedersenBLSUnchainedSwapped() (cs *Scheme) {
-	var Pairing = bls.NewBLS12381Suite()
+	var Pairing = bls.NewBLS12381SuiteWithDST(bls.DefaultDomainG2(), nil)
```

Comment:
Related to another comment, can we explicitely name the domains here, so it's clear why this one is buggy and the other one is not ?